contracts/src/token/erc20/extensions/erc4626.rs

```diff
@@ -1143,11 +1143,11 @@ impl Erc4626 {
     }
 }
 
-// TODO: Add missing tests once `motsu` supports calling external contracts.
 #[cfg(all(test, feature = "std"))]
```

Comment:
```suggestion
// TODO: Add missing tests once `motsu` supports calling external contracts.
#[cfg(all(test, feature = "std"))]
```
No new tests were added, so this comment should be kept